models/birthday.py

```diff
@@ -0,0 +1,11 @@
+from datetime import datetime
+
+from sqlalchemy.orm import Mapped, mapped_column
+
+from models.models import Base
+
+
+class Birthday(Base):
+    __tablename__ = "birthdays"
+    date: Mapped[datetime] = mapped_column(primary_key=True)
+    age: Mapped[int] = mapped_column(primary_key=True)
```

Comment:
do you need to store the age here, or can you just count the number of birthdays?
Also may be worth adding a column for which user did the happy birthday message.

Reply:
spoke with owen about this in tech-team, just makes life slightly easier. Good idea for the user.